taxi_service/settings.py

```diff
@@ -56,7 +56,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": [BASE_DIR, "templates"],
```

Comment:
The `DIRS` setting in the `TEMPLATES` configuration correctly includes the `BASE_DIR` and `templates` directory, ensuring that Django knows where to look for template source files.